src/Analysis/Engine/Impl/Values/SpecializedNamespace.cs

```diff
@@ -217,6 +217,15 @@ abstract class SpecializedNamespace : AnalysisValue {
             }
         }
 
+        public override string Name {
+            get {
```

Comment:
This can be written with expression:
```cs
public override string Name => _original != null ? _original.Name : base.Name;
```

Reply:
Probable even
```cs
public override string Name => _original?.Name ?? base.Name;
```

Reply:
No, that's a bit different semantics. In original code, if `_original != null` and `_original.Name == null`, it will return `null`. In case of `_original?.Name ?? base.Name`, it will return `base.Name`. Can't say what is the right behavior though.